lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err, aclObject) {});
+ * myFile.acl.makePrivate(function(err, aclObject) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
+};
+
+/**
+ * Make a {module:storage/bucket} or {module:storage/file} object publicly
+ * readable.
+ *
+ * This is a short-hand method which will grant "reader" permissions to the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePublic
+ *
+ * @example
+ * myBucket.acl.makePublic(function(err, aclObject) {});
+ * myFile.acl.makePublic(function(err, aclObject) {});
+ */
+Acl.prototype.makePublic = function(callback) {
+  this.add({
+    scope: 'allUsers',
+    role: 'READER'
+  }, callback);
+};
+
+/**
  * Transform API responses to a consistent object format.
  *
  * @private
  */
 Acl.prototype.makeAclObject_ = function(accessControlObject) {
   var obj = {
-    scope: accessControlObject.scope,
+    scope: accessControlObject.entity,
```

Comment:
The API never changed, it was always `entity` ([see here](https://github.com/google/google-api-nodejs-client/tree/master/apis/storage)) so I'm not sure where `scope` came from, may have been a simplification of the terminology which makes sense to me. For upstream support it makes sense to change to `entity` now that we have more reasonable methods to interacting with ACLs, people who want to do it manually can stick to the docs more closely.